$OpenBSD: patch-Source_JavaScriptCore_llint_LowLevelInterpreter_cpp,v 1.1 2013/04/13 09:11:02 landry Exp $
https://bugs.webkit.org/show_bug.cgi?id=103128
http://trac.webkit.org/changeset/145551 fixes missing Double2Ints on ppc
https://bugs.webkit.org/show_bug.cgi?id=113638
tentative fix for ppc/sparc64/mips64el - LLIntAssembly.h uses isnan() without including math.h (or cmath?)
--- Source/JavaScriptCore/llint/LowLevelInterpreter.cpp.orig	Tue Mar 26 19:25:12 2013
+++ Source/JavaScriptCore/llint/LowLevelInterpreter.cpp	Sun Mar 31 11:47:48 2013
@@ -117,6 +117,17 @@ static double Ints2Double(uint32_t lo, uint32_t hi)
     u.ival64 = (static_cast<uint64_t>(hi) << 32) | lo;
     return u.dval;
 }
+
+static void Double2Ints(double val, uint32_t& lo, uint32_t& hi)
+{
+    union {
+        double dval;
+        uint64_t ival64;
+    } u;
+    u.dval = val;
+    hi = static_cast<uint32_t>(u.ival64 >> 32);
+    lo = static_cast<uint32_t>(u.ival64);
+}
 #endif // USE(JSVALUE32_64)
 
 } // namespace LLint
